lms/djangoapps/grades/models.py

```diff
@@ -663,6 +663,7 @@ def update_or_create(cls, user_id, course_id, **kwargs):
 
         cls._emit_grade_calculated_event(grade)
         cls._update_cache(course_id, user_id, grade)
+        cls._emit_persistent_grade_summary_changed_event(course_id, user_id, grade)
```

Comment:
Could we put **openedx** to the name of the function? in this way, we can differentiate between tasks that are linked to internal processes of the platform and procedures that are associated with external processes (in this case openedx-events). I propose that it looks like this **_emit_openedx_persistent_grade_summary_changed_event**

Reply:
Yes, I think so, I have already made the change, thank you for your advice.